### PR TITLE
Update to Swift 5

### PR DESCRIPTION
**Pull request by mckaskle**
This also prepares for a new release: `0.6.0`.

### CODE REVIEW

**Comment by mckaskle** [comment]
@louoso I'm running into some issues w/ the tests. This is one example:

```
    func testFloatLabelElement() {
        let app = XCUIApplication()
        let elementsQuery = app.scrollViews.otherElements
        let floatLabelTextField = elementsQuery.textFields["floatLabel"]
        
        floatLabelTextField.tap()
        app.typeText("Hello World")
        app.alerts.buttons["Dismiss"].tap()
    }
```

The issue is that the alert shows after the "H" from "Hello World" is typed and then the simulator hangs because it's still trying to type "Hello World" but the text field is no longer the first responder. There are a few ways I could fix the tests:

1. Only type a single character instead of "Hello World".
2. Debounce value changes for some amount of time so that "Hello World" can be completely typed out before the alert is shown.
3. Only call the value change observer once the text field ends editing.

I don't know what the original behavior was, but I'm guessing 1 is the way to go. It seems like the right approach and would also be an easy fix. I'm guessing 2 & 3 would be breaking changes.

File: Carthage/Checkouts/stackviewcontroller/README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# StackViewController
-[![Circle CI](https://circleci.com/gh/seedco/StackViewController/tree/master.svg?style=svg)](https://circleci.com/gh/seedco/StackViewController/tree/master)
```
Comment (louoso) [inline Carthage/Checkouts/stackviewcontroller/README.md]: Why remove the Circle CI badge?

File: Carthage/Checkouts/stackviewcontroller/StackViewController/StackViewContainer.swift [hunk 2]
```diff
@@ -13,13 +13,13 @@ import UIKit
 /// using view controller composition to display content.
 open class StackViewContainer: UIView, UIScrollViewDelegate {
     /// The scroll view that is the superview of the stack view.
-    open let scrollView: AutoScrollView
+    public let scrollView: AutoScrollView
```
Comment (louoso) [inline Carthage/Checkouts/stackviewcontroller/StackViewController/StackViewContainer.swift]: Is this technically a breaking API change? If so, should we increment the pod version to reflect?

File: Carthage/Checkouts/stackviewcontroller/README.md (hunk 1 above)
Comment (mckaskle) [reply Carthage/Checkouts/stackviewcontroller/README.md]: Here's the commit that changed that: https://github.com/seedco/StackViewController/commit/6649a15141c397d625b842f384351e0fdc547f34#diff-04c6e90faac2675aa89e2176d2eec7d8

**Pull request merged**